server/data/ciagInductionClient.ts

```diff
@@ -12,7 +12,7 @@ export default class CiagInductionClient {
     token: string,
   ): Promise<CiagInductionSummaryListResponse> {
     const requestBody: GetCiagInductionSummariesRequest = { offenderIds: prisonNumbers }
-    return CiagInductionClient.restClient(token).post({
+    return CiagInductionClient.restClient(token).post<CiagInductionSummaryListResponse>({
```

Comment:
All the client classes that use the `restClient` can now use type generics in their method calls 👍